ide/eclipse/bundles/org.apache.syncope.ide.eclipse.plugin/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,48 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Plugin
+Bundle-SymbolicName: org.apache.syncope.ide.eclipse.plugin;singleton:=true
+Bundle-Version: 2.0.0.qualifier
```

Comment:
* is 'qualifier' required?
* I guess this must remain static "2.0.0" and could not be derived from `${syncope.version}`, right?